src/test/java/com/teradata/tpcds/CustomerAddressGeneratorTest.java

```diff
@@ -52,118 +52,118 @@ public void testScaleFactor100()
     {
         Session session = TEST_SESSION.withScale(100).withParallelism(100).withChunkNumber(1);
         ChunkBoundaries chunkBoundaries = splitWork(CUSTOMER_ADDRESS, session);
-        assertPartialMD5(chunkBoundaries.getFirstRow(), chunkBoundaries.getLastRow(), CUSTOMER_ADDRESS, session, "3b94a24f194bd74cb4f42a1763613d1b");
+        assertPartialMD5(chunkBoundaries.getFirstRow(), chunkBoundaries.getLastRow(), CUSTOMER_ADDRESS, session.withChunkNumber(1), "3b94a24f194bd74cb4f42a1763613d1b");
```

Comment:
these tests don't need to change because we set the session with the chunk number before each test.